awscli/clidriver.py

```diff
@@ -77,6 +81,7 @@ def __init__(self, session=None):
         self._cli_data = None
         self._command_table = None
         self._argument_table = None
+        self._command_table_with_aliases = None
```

Comment:
Do we need to have both forms of command tables with/without aliases?  It would seems simpler to just update the existing command table, but I might be missing something here (is it related to docs?)